packages/babel-jest/src/index.js

```diff
@@ -129,7 +134,17 @@ const createTransformer = (options: any) => {
 
       // babel v7 might return null in the case when the file has been ignored.
       const transformResult = babelTransform(src, theseOptions);
-      return transformResult ? transformResult.code : src;
+
+      if (!transformResult) {
+        return src;
+      }
+
+      const shouldReturnCodeOnly =
```

Comment:
I think this condition is wrong.

Should be 

```js
const shouldReturnCodeOnly = !transformOptions || transformOptions.returnSourceString == null || transformOptions.returnSourceString;
```

Reply:
so that if `transformOptions` is not passed in `shouldReturnCodeOnly  === true`

Reply:
potentially

```js
const shouldReturnCodeOnly =
  transformOptions == null ||
  transformOptions.returnSourceString == null ||
  transformOptions.returnSourceString === true;
```

to be explicit

Reply:
Waiting for this change before merging.

Reply:
I'm outside. I'll change n push in an hour.

Reply:
This is pushed!

Reply:
we should have a test for this - mind adding one?

Reply:
Okay. Will add and push.

Reply:
Done!

Reply:
Done!